src/wp-includes/abilities-api.php

```diff
@@ -0,0 +1,170 @@
+<?php
+/**
+ * Abilities API
+ *
+ * Defines functions for managing abilities in WordPress.
+ *
+ * @package WordPress
+ * @subpackage Abilities_API
+ * @since 6.9.0
+ */
+
+declare( strict_types = 1 );
+
+/**
+ * Registers a new ability using Abilities API.
+ *
+ * Note: Do not use before the {@see 'abilities_api_init'} hook.
+ *
+ * @since 6.9.0
+ *
+ * @see WP_Abilities_Registry::register()
+ *
+ * @param string              $name The name of the ability. The name must be a string containing a namespace
+ *                                  prefix, i.e. `my-plugin/my-ability`. It can only contain lowercase
+ *                                  alphanumeric characters, dashes and the forward slash.
+ * @param array<string,mixed> $args An associative array of arguments for the ability. This should include
```

Comment:
https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#1-1-parameters-that-are-arrays